### PR TITLE
CSB error fix, added atomic-field-condition to display language badge

### PR DESCRIPTION
**Pull request by karoomoogon-coveo**
Jira: https://coveord.atlassian.net/browse/DOC-12592

The errors was being caused by atomic-result-badge component used to display the language badge. It appeared to be using the field property to detect whether the language field was defined.
It was replaced by an atomic-field-condition component.

Tested locally and on sandbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -182,12 +182,14 @@
                         id="youtubevideo-ted-ed"
                       ></atomic-result-badge>
                     </atomic-field-condition>
-                    <atomic-result-badge
-                      id="language-badge"
-                      field="language"
-                      icon="https://raw.githubusercontent.com/Rush/Font-Awesome-SVG-PNG/master/black/svg/language.svg"
-                    ></atomic-result-badge>
-                </atomic-result-section-badges>
+                    <atomic-field-condition class="field" if-defined="language">
```
Comment (jpmarceau) [inline index.html]: What is `class="field"` for ? Seems like it wasn't there before?
I also don't see that in the ref docs https://docs.coveo.com/en/atomic/latest/reference/result-template-components/atomic-field-condition/#properties

Comment (karoomoogon-coveo) [reply index.html]: It was always there and is also included in all other similar cases where such conditional are used, even in the ui-kit repo. Although, I just tested it, and if removed, it does not affect anything.

**Review by npushkarskii — APPROVED** [review]
Great job, Krishna! I've forked the sandbox example, added your suggestions there and there's no errors 👍

Before merging, we only need to address a note that JP made about `class="field"`. 🙏

**Review by jpmarceau — APPROVED** [review]
Looks good 👍 
Could you also add the organization endpoint support while you're there? Seems like I forgot it in this branch 🙈

**Pull request merged**